frontend/public/locales/ja/cluster-settings.json

```diff
@@ -1,48 +1,50 @@
 {
-  "Invalid cluster version": "無効なクラスターバージョン",
-  "Cancel update": "更新をキャンセル",
-  "Available updates": "利用可能なアップデート",
-  "Failing": "失敗する",
-  "View conditions": "条件を見る",
-  "Update to {{version}} in progress": "に更新{{version}}進行中",
+  "Invalid cluster version": "無効なクラスターのバージョン",
+  "Cancel update": "更新のキャンセル",
+  "Available updates": "利用可能な更新",
+  "Failing": "失敗",
+  "View conditions": "状態の表示",
+  "Update to {{version}} in progress": "{{version}} への更新が進行中",
   "No update channel selected": "更新チャネルが選択されていません",
   "Version not found": "バージョンが見つかりません",
   "Error retrieving": "取得エラー",
-  "Up to date": "最新の",
-  "View release notes": "リリースノートを見る",
+  "Up to date": "最新",
+  "View release notes": "リリースノートの表示",
   "Unknown": "Unknown",
```

Comment:
"Unknown" could be translated into Japanese.

Reply:
They added a translation for this.